README.md

```diff
@@ -23,7 +23,8 @@ the codebase can drastically change at any time. Also note that some parts of th
 'shiftable' yet, so modifying them could be difficult at this point.
 ```
 
-This is a WIP decompilation of The Legend of Zelda: Ocarina of Time.
+This is a WIP decompilation of The Legend of Zelda: Ocarina of Time. **It is not a PC port.** It has never been a PC port. There will never be a PC port in this repository. Please do not claim that this project is a PC port. Anything claiming this is a PC port is misinformed. For more information you can get in touch with the team on our [Discord server](https://discord.zelda64.dev).
```

Comment:
I don't like this wording as it sounds needlessly whiny. It would be better if we clarified the end goal a bit better.
```suggestion
This is a decompilation of The Legend of Zelda: Ocarina of Time for the N64 and in the future, the iQue. For information on the PC Port, [click here](https://www.youtube.com/watch?v=yPYZpwSpKmA)
```